packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -203,6 +203,7 @@ const configureRequest = async (
         interpolateVars(requestCopy, envVars, collectionVariables, processEnvVars);
         const { data: authorizationCodeData, url: authorizationCodeAccessTokenUrl } =
           await resolveOAuth2AuthorizationCodeAccessToken(requestCopy, collectionUid);
+        request.method = 'POST';
```

Comment:
The request here is just the token endpoint request, right? I am not overwriting the user's collection request, I hope?